rows.c

```diff
@@ -0,0 +1,24 @@
+/* Matthew Danielson
```

Comment:
liked that you asked the user for input, but your code didn't work quite so well. Your program missed the line before the middle row for every input.